gammapy/modeling/models/spatial.py

```diff
@@ -606,6 +627,11 @@ class DiskSpatialModel(SpatialModel):
     phi = Parameter("phi", "0 deg", frozen=True)
     edge_width = Parameter("edge_width", value=0.01, min=0, max=1, frozen=True)
 
+    @property
+    def evaluation_bin_size_min(self):
+        """ Minimal evaluation bin size (`~astropy.coordinates.Angle`)."""
+        return self.r_0.quantity * (1 - self.edge_width.quantity)
```

Comment:
I used a similar condition that for the evaluation radius but with a subtraction instead of addition. However I think that the disk is well defined as long as the binning is smaller or egal to r_0.

Reply:
I think an approximate choice is fine but maybe keep it conservative for now...it can be optimised for performance later with a dedicated study.